moveit_kinematics/CMakeLists.txt

```diff
@@ -7,8 +7,9 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost)
-find_package(orocos_kdl)
+find_package(Boost REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(orocos_kdl REQUIRED)
```

Comment:
by finding these three as system deps, I believe you need to add them after the catkin_package DEPENDS clause, if they are used in any of the public-facing headerfiles:
```
catkin_package(
...
DEPENDS
  Boost
  Eigen3
  orocos_kdl
)
```

Reply:
Probably you are right. This was out-of-the-scope of this PR.

Reply:
Actually, it wasn't like this, because the plugins don't expose their headers publicly.

Reply:
Ah, then that makes sense! Thanks for the clarification.